xpano/algorithm/stitcher_pipeline.h

```diff
@@ -40,6 +40,7 @@ struct StitcherData {
 
 struct StitchingResult {
   int pano_id = 0;
+  int export_code;
```

Comment:
Similarly to the first comment, I think you can put cv::Stitcher::Status here directly and drop the optional from the pano member.